README.md

```diff
@@ -1,24 +1,11 @@
-# README
+# Little Shop
```

Comment:
i suggest adding an overview of what this application does here and perhaps how to pull it down and get it working locally 👍